VIDEO_6_SCRIPT.md

```diff
@@ -0,0 +1,53 @@
+# Video 6: Working in community
+
+- What and why working within a community
+- AsyncAPI and its community
+- How to join the AsyncAPI community
+
+Hello everybody! I hope that you’re doing well. Welcome again to the AsyncAPI video, where we explain the project and its progress. Today we’ll talk about the AsyncAPI community.
+
+## What and why working within a community
```

Comment:
you should try this text as a potential lector for the video 😛 

it has many difficult words, talks about many nice things but I have the impression it is overcomplicating the message 🤔 

why not, in super simple words 😄 , just write "that community is about gathering people with the same goals and vision to work as equals together, collaborate and drive the change at scale much grater that they could as individuals"?

it is to serious now, not good for video I think